frontend/src/components/ResourceActionsColumn.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
+import { Button, Timestamp, Tooltip } from '@patternfly/react-core';
+import { BanIcon } from '@patternfly/react-icons';
+import { ActionsColumn } from '@patternfly/react-table';
+
+type Props = React.ComponentProps<typeof ActionsColumn> & {
+  resource: K8sResourceCommon;
+};
+
+const ResourceActionsColumn: React.FC<Props> = ({ resource, ...props }) =>
```

Comment:
I think we should have a storybook component with our own table and this components, not only targeting this to kserve only (not saying this for this PR, i was thinking that if we want this implementation in other pages, I think storybook could be great for our custom components)